blog/content/post/2023-01-23-3mdeb-insights-winter-2023.md

```diff
@@ -0,0 +1,122 @@
+---
+title: 3mdeb Insights - Winter 2023
+abstract: 'This blog will provide an in-depth look at upcoming plans and events
```

Comment:
@Asiderr we should extend the abstract to have at least 2-3 sentences

Reply:
Added more sentences to the abstract. 083a71e